### ISSUE TITLE
PHP version outdated

### ISSUE THREAD

**Issue by enrico69** [ryanstreet/magento2-vagrant#15]
Hello everybody

Had a little problem with the required version of PHP for the Magento package.
I had no error when I used these commands:

> git submodule init
> git submodule update

But I noticed that the Magento folder was anormally filled. So I launched manually a composer update, and get this message:

> Thispackage requires php ~5.5.22|~5.6.0|~7.0.0 but your PHP version (5.5.12) does not satisfy that 
> requirement.

Has a quick fix, I had to change the line # 11 of the composer.json file.

Regards

**Comment by christian-walker** [comment]
Hello,

What change did you make on line # 11?  I changed the php version to 5.5.9 and was able to install composer, BUT Magento still didn't allow me to set up the installation because my server php version is on 5.5.9.  Is there a way to upgrade to 5.6?

**Comment by enrico69** [comment]
Hi!

As a dirty workaround I simply replaced by require php 5.5.
It was only to be able to go further in my tests, but it is clearly postponing problems to later because the dependencies need the specified version of PHP which was originaly given in the json file. At the end, I was stucked at the same level you are.

If you are interested on, I made a fork of this repo, and replaced PHP 5.5 by PHP 7.0 + added phpMyAdmin:

[https://github.com/enrico69/magento2-vagrant](https://github.com/enrico69/magento2-vagrant)
